client/v1/client.go

```diff
@@ -2282,3 +2289,107 @@ func (d *Duration) UnmarshalText(data []byte) error {
 	*d = Duration(dur)
 	return nil
 }
+
+// TODO: this is also used in kapacitor/main.go
```

Comment:
Old comment. Previously this code was in `kapacitor/main.go` and unexported. I moved it here and exported it. The `TODO` item was meant to remind me to update `kapacitor/main.go` to use this code instead.